gnucash/report/standard-reports/budget.scm

```diff
@@ -262,10 +260,6 @@
      (gnc:make-simple-boolean-option
       gnc:pagename-display optname-show-totalcol
       "s4" opthelp-show-totalcol #f))
-    (add-option
-     (gnc:make-simple-boolean-option
```

Comment:
Instead of deleting option I think it's safer to change this to `(gnc:make-internal-option)` so that saved-reports will still load. And add comment to explain this option was never used.

Reply:
I've made the change, but I'm not sure if it is the right way. I'm not familiar at all with Scheme and all the internals of reports

Reply:
Your scheme skills are not that bad :) you're right "__" in section will hide it, but we want to use `(gnc:make-internal-option gnc:pagename-display "Roll up budget amounts to parent" #f)` instead because options are found via section/name strings, and saved-reports will try to restore options by searching them. (I'd suggest `#f` because in scheme it is conventionally used for all NULL/unused/missing data). So it should be something like:

```scheme
 (add-option
  (gnc:make-internal-option
   gnc:pagename-display "Roll up budget amounts to parent" #f))
```

Reply:
Well I'll need to put foot from mouth.
It looks like removing an option is actually harmless.
Apologies for misinformation.

Reply:
No problem.
So now I reverted back, and completely removed the option.
This leaves a cleaner code